test/vm.install

```diff
@@ -46,8 +46,10 @@ if [ -d /var/tmp/debian ]; then
     systemctl disable docker.service io.podman.service
 fi
 
-# HACK: avoid leaking processes with podman < 2.0.4; see https://github.com/containers/podman/commit/2b6dd3fb4384
-sed -i '/KillMode=process/d; s/Type=oneshot/Type=simple/' /lib/systemd/user/podman.service /lib/systemd/system/podman.service
+if rpm -q podman >/dev/null 2>&1; then
```

Comment:
Why not using 'else' branch? Seems a bit clearer but just nitpick of course :)

Reply:
You mean for "Debian vs. not"? No particular reason, I suppose we don't support a third package format right now. The hack should disappear completely in a week anyway, though.

Reply:
Yeah, as the condition is there just to avoid breaking debian. But fair enough since it is rather tmp thing :)